src/Microsoft.Data.SqlClient/tests/tools/Microsoft.Data.SqlClient.TestUtilities/config.default.json

```diff
@@ -17,7 +17,8 @@
     "AzureKeyVaultClientSecret": "",
     "SupportsIntegratedSecurity": true,
     "LocalDbAppName": "",
-    "LocalDbSharedInstanceName":"",
+    "LocalDbSharedInstanceName": "",
+    "LocalDbNamedPipeSet": false,
```

Comment:
Let's be consistent with the other keys and use either `LocalDbNamedPipesEnabled` or `SupportsLocalDbNamedPipes`.